<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>490</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>730</width>
    <height>490</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>730</width>
    <height>490</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="Dtk::Widget::DFrame" name="columnDfrm">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>730</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="dockerListWdg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>95</y>
     <width>730</width>
     <height>395</height>
    </rect>
   </property>
  </widget>
  <widget class="Dtk::Widget::DFrame" name="conDfrm">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>45</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Dtk::Widget::DFrame</class>
   <extends>QFrame</extends>
   <header>DFrame</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
